lady_luck.js

```diff
@@ -174,19 +152,17 @@ function nwodToText(outcome) {
 
 	var newResults = outcome.results;
 
-	for (var i = 0; i < newResults.length; i++) {
-		if (newResults[i] >= outcome.again) {
-			newResults[i] = "**_" + newResults[i] + "_**";
-		} else if (newResults[i] >= 8) {
-			newResults[i] = "**" + newResults[i] + "**";
-		} else if (outcome.botching && newResults[i] == 1) {
-			newResults[i] = "~~1~~";
-		}
-	}
+	for (const i in newResults)
+		if (newResults[i] >= outcome.again)
+			newResults[i] = `**_${newResults[i]}_**`;
+		else if (newResults[i] >= 8)
+			newResults[i] = `**${newResults[i]}**`
+		else if (outcome.botching && newResults[i] == 1)
+			newResults[i] == '~~1~~'
 
 	notes = [];
 	if (outcome.again == 8 || outcome.again == 9) {
-		notes.push(outcome.again + "-Again");
+		notes.push(`${outcome.again}-Again`);
```

Comment:
Interpolation runs slower than concatenation, and in this case, it doesn't even add any readability advantage.

https://www.measurethat.net/Benchmarks/Show/195/1/string-interpolation-vs-concatenation#latest_results_block